src/accordion.ts

```diff
@@ -97,8 +97,14 @@ export default class Accordion extends LitElement {
                 opacity: [0, 1],
               },
               {
-                duration: 150,
-                easing: 'ease-in',
+                duration: Number(
+                  getComputedStyle(this.#defaultSlotElementRef.value)
+                    .getPropertyValue('--private-open-duration')
+                    .replace('ms', ''),
```

Comment:
Our durations use `ms`, but `duration` here must be a number.  So we lob it off.

Otherwise you get:

```bash
accordion.ts?t=1752844652129:100 Uncaught TypeError: Failed to execute 'animate' on 'Element': duration must be non-negative or auto
    at set open (accordion.ts?t=1752844652129:100:58)
    at Final.onSummaryClick_fn (accordion.ts?t=1752844652129:214:13)
```